From c772f33116a08d5e6ea66f6f5048f460588da0bb Mon Sep 17 00:00:00 2001
From: Fabian Kosmale <fabian.kosmale@qt.io>
Date: Mon, 6 Oct 2025 14:15:02 +0200
Subject: [PATCH 143/239] CRA: Review qmltoolingsettings

Mark everything as (only) significant, the actual parsing logic is in
QSettings.

Pick-to: 6.8
Fixes: QTBUG-136207
QUIP: 23
Change-Id: Ia9bfc734707153a3f31ac2cf5afe04b57faa3508
Reviewed-by: Sami Shalayel <sami.shalayel@qt.io>
(cherry picked from commit 2ebc035e380fb9c62ecc09b8c1f210a9c24191ff)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qmltoolingsettings/qqmltoolingsettings.cpp | 1 +
 src/qmltoolingsettings/qqmltoolingsettings_p.h | 1 +
 src/qmltoolingsettings/qqmltoolingutils.cpp    | 1 +
 src/qmltoolingsettings/qqmltoolingutils_p.h    | 1 +
 4 files changed, 4 insertions(+)

diff --git a/src/qmltoolingsettings/qqmltoolingsettings.cpp b/src/qmltoolingsettings/qqmltoolingsettings.cpp
index bc746f89ef..a222ba29ef 100644
--- a/src/qmltoolingsettings/qqmltoolingsettings.cpp
+++ b/src/qmltoolingsettings/qqmltoolingsettings.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant
 
 #include "qqmltoolingsettings_p.h"
 
diff --git a/src/qmltoolingsettings/qqmltoolingsettings_p.h b/src/qmltoolingsettings/qqmltoolingsettings_p.h
index 2ceab4727b..2dbc67ce4f 100644
--- a/src/qmltoolingsettings/qqmltoolingsettings_p.h
+++ b/src/qmltoolingsettings/qqmltoolingsettings_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant
 
 #ifndef QQMLTOOLINGSETTINGS_P_H
 #define QQMLTOOLINGSETTINGS_P_H
diff --git a/src/qmltoolingsettings/qqmltoolingutils.cpp b/src/qmltoolingsettings/qqmltoolingutils.cpp
index 1cf1602b4b..ecca0a1cbd 100644
--- a/src/qmltoolingsettings/qqmltoolingutils.cpp
+++ b/src/qmltoolingsettings/qqmltoolingutils.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant
 
 #include "qqmltoolingutils_p.h"
 
diff --git a/src/qmltoolingsettings/qqmltoolingutils_p.h b/src/qmltoolingsettings/qqmltoolingutils_p.h
index 877d490355..b914fd3dbb 100644
--- a/src/qmltoolingsettings/qqmltoolingutils_p.h
+++ b/src/qmltoolingsettings/qqmltoolingutils_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant
 
 #ifndef QQMLTOOLINGUTILS_P_H
 #define QQMLTOOLINGUTILS_P_H
-- 
2.51.2

